examples-gae/gae-jdk7/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.2-bin.zip
```

Comment:
This is a different version than used elsewhere.

Reply:
I must have copied this from the GAE examples repo. Fixed.